### PR TITLE
Add Chattermill to keda users

### PR DESCRIPTION
**Pull request by ivaravko**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -344,6 +344,10 @@ logo = "schreder-hyperion.png"
 url = "https://www.teamleader.eu"
 logo = "teamleader.png"
 
+[[params.users]]
+url = "https://chattermill.com"
+logo = "chattermill.png"
```
Comment (tomkerkhove) [inline config.toml:349]: Can you alphabetically sort please?

**Review by tomkerkhove — APPROVED** [review]
Thank you for the PR and using KEDA!

File: config.toml (hunk 1 above)
Comment (ivaravko) [reply config.toml]: Done

**Comment by tomkerkhove** [comment]
Thank you @ivaravko!

**Comment by tomkerkhove** [comment]
@ivaravko We are looking for KEDA end-users who want to be interviewed by CNCF for our CNCF Graduation proposal.

Are you willing to share about your experiences on using KEDA? This can be anonymously if need be.

If so, feel free to fill in https://forms.gle/91dD4RVxifWsS2hc7

**Pull request merged**